code-challenges.test.js

```diff
@@ -0,0 +1,159 @@
+// ASSESSMENT 2: Coding Practical Questions with Jest
+
+// Please read all questions thoroughly
+// Pseudo coding is REQUIRED
+// If you get stuck, please leave comments to help us understand your thought process
+
+// Use test driven development to complete the following questions
+// Add appropriate dependencies: $ yarn add jest
+
+// Reminder: The test will call your function
+// Run the file with the following command: $ yarn jest
+
+// --------------------INSTRUCTOR EXAMPLE: Create a function that takes in an array of numbers and returns an array with all the numbers multiplied by 3.
+
+// Pseudo code:
+
+// a) Create a test with expect statements for each of the variables provided.
+
+
+
+
+const numbersArray1 = [6, 7, 8, 9, 10]
+// Expected output: [18, 21, 24, 27, 30]
+const numbersArray2 = [24, 27, 30, 33, 36]
+// Expected output: [72, 81, 90, 99, 108]
+
+// b) Create the function that makes the test pass.
+
+// --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
+
+// a) Create a test with expect statements for each of the variables provided.
+
+const object1 = { number: 15 }
+// Expected output: "15 is divisible by three"
+const object2 = { number: 0 }
+// Expected output: "0 is divisible by three"
+const object3 = { number: -7 }
+// Expected output: "-7 is not divisible by three"
+
+// b) Create the function that makes the test pass.
+
+// Pseudo code:
+
+// Write a describe statement that takes in the argument of a string and function.
+
+// Write an it statement in the describe block as a string that describe what the test is doing with a function 
+
+// Write 3 expect statements that take in the each encased function call with the .toEqual() method with strings encased in the .toEqual method
+
+// Create a function with a conditional statement 
+// - if the object is divisible by 3 then it'll return that number is divisible by 3
+// - if else, it'll return the number is not divisible by 3
```

Comment:
Expand on this pseudo code. You said if the object is divisible by 3, but you can't perform arithmetic on an object. Then you say that it'll return that number, when you had previously said object.
How are you accessing the value in the object? What tools are you going to use for the logic?